test/e2e/node_drain.go

```diff
@@ -233,6 +233,31 @@ func NodeDrainTimeoutSpec(ctx context.Context, inputGetter func() NodeDrainTimeo
 				WaitForDeploymentAvailableInterval: input.E2EConfig.GetIntervals(specName, "wait-deployment-available"),
 			})
 		}
+		By("Deploy an unevictable Deployment with `wait-completed` label.")
```

Comment:
I think in general the test can work roughly as is.

But as we are now assuming order 0 for WaitCompleted Pods we have to change the order of some other Pods to make sure they can be drained (currently the drain of these Pods is blocked by the waitcompleted Pods).

tl;dr I think the Pods with drain order 1 should get order -5, the Pods with drain order 5 should get order -1

Then all of these Pods can be evicted / drained

Reply:
done, ptal